server.js

```diff
@@ -43,6 +44,7 @@ app
     .get("/log-out", logout)
 
     .post("/", createAccount)
+    .post("/matches", matches)
```

Comment:
i think the path should be something like "/liked" because your feature is about liking other users and display the liked persons on a seperate page. Also, since we adhere to the standard that you give your function a name that describes it's purpose you should rename it to something like "renderLikedPersons" where it renders a "liked-persons.ejs" file/page where the liked persons get displayed.
```suggestion
    .post("/liked", renderLikedPersons)
```